_overviews/scala3-macros/tutorial/inline.md

```diff
@@ -379,17 +379,17 @@ It is also possible to create _abstract inline definitions_.
 trait InlineLogger:
   inline def log(inline x: Any): Unit
 
-class PrintLogger inline InlineLogger:
+class PrintLogger extends InlineLogger:
   inline def log(inline x: Any): Unit = println(x)
 ```
 
 This forces the implementation of `log` to be an inline method and also allows `inline` parameters.
 Counterintuitively, the `log` on the interface `InlineLogger` cannot be directly called. The method implementation is not statically known and we thus do not know what to inline.
-Calling an abstract inline methods thus results in an error.
+Calling an abstract inline method thus results in an error.
 The usefuleness of abstract inline methods becomes apparent when used in another inline method:
 
 ```scala
-inline def logged(logger: Logger, x: Any) =
+inline def logged(logger: InlineLogger, x: Any) =
```

Comment:
I'm not sure here but 'Logger' does not appear in this example, and this way we actually call the method on 'InlineLogger', that is we call the _abstract method_